packages/vaadin-grid/test/basic.test.js

```diff
@@ -1,14 +1,17 @@
 import { expect } from '@esm-bundle/chai';
 import sinon from 'sinon';
 import { aTimeout, fixtureSync } from '@vaadin/testing-helpers';
-import { afterNextRender } from '@polymer/polymer/lib/utils/render-status.js';
 import {
   flushGrid,
   getBodyCellContent,
   getCell,
   getCellContent,
   getFirstVisibleItem,
+  getLastVisibleItem,
+  getPhysicalAverage,
+  getPhysicalItems,
   infiniteDataProvider,
+  scrollGrid,
   scrollToEnd
 } from './helpers.js';
 import '@polymer/iron-list/iron-list.js';
```

Comment:
This import can be removed, as well as the `@polymer/iron-list` dev dependency in `package.json`.

Reply:
Done